src/css/base.scss

```diff
@@ -4324,7 +4324,7 @@ textarea::placeholder {
     display: -webkit-box;
     -webkit-box-orient: vertical;
     -webkit-line-clamp: 3;
-    word-break: break-all;
+    word-break: break-word;
```

Comment:
If this is asked by product, should be done for others as well

Reply:
They only asked me to implement this here